mod-pubsub-server/src/main/java/org/folio/dao/EventDescriptorDao.java

```diff
@@ -54,7 +54,7 @@
    * Deletes {@link EventDescriptor} by event type
    *
    * @param eventType eventDescriptor event type
-   * @return future with true of succeeded
+   * @return a future completed with a result
```

Comment:
What does it mean "with a result"? Maybe we can change it to "succeeded future if exactly one row was updated, failed future otherwise"

Reply:
fixed